CHANGES

```diff
@@ -9,6 +9,26 @@
 
  Changes between 1.1.1 and 3.0.0 [xx XXX xxxx]
 
+  *) Deprecated low level ECDH and ECDSA functions. These include:
+         ECDH_compute_key, ECDSA_SIG_new, d2i_ECDSA_SIG, i2d_ECDSA_SIG,
+         ECDSA_SIG_get0, ECDSA_SIG_get0_r, ECDSA_SIG_get0_s, ECDSA_SIG_set0,
+         ECDSA_do_sign, ECDSA_do_sign_ex, ECDSA_do_verify, ECDSA_sign_setup,
+         ECDSA_sign, ECDSA_sign_ex, ECDSA_verify and ECDSA_size.
```

Comment:
I kind of think that perhaps the `ECDSA_SIG_*` functions should stay? They are probably still useful even after the move to providers if for some reason you want do dive into the r and s values. I had a question just the other day where someone had the raw r and s values and wanted to construct a signature that was readable by OpenSSL with them.

Reply:
I am definitely +1 to this as this is an interface to access the data and not to invoke any crypto operations.